uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/PublicHolidayLeaveRequestDeletion.php

```diff
@@ -46,8 +46,10 @@ public function deleteForAllContacts(PublicHoliday $publicHoliday) {
    *
    * @param int $contactID
    * @param \CRM_HRLeaveAndAbsences_BAO_PublicHoliday $publicHoliday
+   * @param boolean $softDelete
+   *   If true, the public holiday request will be soft deleted
    */
-  public function deleteForContact($contactID, PublicHoliday $publicHoliday) {
+  public function deleteForContact($contactID, PublicHoliday $publicHoliday, $softDelete = FALSE) {
     $leaveRequest = LeaveRequest::findPublicHolidayLeaveRequest($contactID, $publicHoliday);
```

Comment:
The comment I left about the `findPublicHolidayLeaveRequest` method also applies here.